.github/workflows/devcenter-doc-update.yml

```
@@ -0,0 +1,18 @@
+name: Update CLI Command DevCenter Documentation
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+jobs:
+  update-devcenter-command-docs:
+    name: Update Devcenter command docs
+    runs-on: pub-hk-ubuntu-22.04-small
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install package deps
```

I will suggest adding the required `setup` actions to ensure (and control) the required tool versions for Node and Ruby.

```suggestion
      - name: Use Node.js 16.x
        uses: actions/setup-node@v3
        with:
          node-version: 16.x
          cache: yarn
      - uses: ruby/setup-ruby@ec02537da5712d66d4d50a0f33b7eb52773b5ed1
        with:
          ruby-version: '3.1'
      - name: Install package deps
```

---

Thanks for this info. Hopefully it just works, but if it doesn't this will be very useful.